### ISSUE TITLE
Asking about the hyperparameters used when N=100 and N=600

### ISSUE THREAD

**Issue by kjiang10** [wohlert/semi-supervised-pytorch#34]
Hi,

First, thank you so much for replicating Semi-supervised Learning with Deep Generative Models (Kingma 2014) in PyTorch!

Currently I am doing some research about semi-supervised VAE and have been trying to reproduce the results based on the **M2** model, i.e., the expected acc would be 88% given N (number of labeled samples) = **100** and 95% given N = **600**. However, no matter how I tuned the hyperparameters (alpha, learning rate, batch size), there always exist an 8% - 12 % difference between the actual accuracy and the expected accuracy. I'm not sure whether I just overlooked some details but would you mind shedding some lights on what hyperparameters you used or some other details I might missed? Thank you.
